From 1b8630672f3458b8354a507f1004ee853199612f Mon Sep 17 00:00:00 2001
From: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Date: Tue, 26 Jun 2018 15:29:12 +0300
Subject: [PATCH 074/107] armv8: fsl-layerscape: add missing debug stream ID

Add a define with a value for the missing debug stream ID.

Signed-off-by: Laurentiu Tudor <laurentiu.tudor@nxp.com>
---
 .../asm/arch-fsl-layerscape/stream_id_lsch2.h      |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/include/asm/arch-fsl-layerscape/stream_id_lsch2.h b/arch/arm/include/asm/arch-fsl-layerscape/stream_id_lsch2.h
index b326fe5..e829111 100644
--- a/arch/arm/include/asm/arch-fsl-layerscape/stream_id_lsch2.h
+++ b/arch/arm/include/asm/arch-fsl-layerscape/stream_id_lsch2.h
@@ -51,6 +51,7 @@
 #define FSL_QDMA_STREAM_ID		7
 #define FSL_EDMA_STREAM_ID		8
 #define FSL_ETR_STREAM_ID		9
+#define FSL_DEBUG_STREAM_ID		10
 
 /* PCI - programmed in PEXn_LUT */
 #define FSL_PEX_STREAM_ID_START		11
-- 
1.7.1

